tests/acceptance/test_data_mappers.py

```diff
@@ -146,7 +152,9 @@ def test_it_creates_without_optionals(
     expected = deepcopy(data_mapper)
     expected["Format"] = "parquet"
     expected["CreatedBy"] = {
-        "Username": "aws-uk-sa-builders@amazon.com",
+        "Username": "aws-uk-sa-builders@amazon.com"
+        if stack["AuthMethod"] != "IAM"
+        else "N/A",
```

Comment:
this is quite a significant change as CreatedBy is used for things like audit. Is there no way to determine who has done what when IAM is used?

Reply:
Good point, i'll investigate options for this....